_data/remote.yml

```diff
@@ -15,6 +15,14 @@ websites:
       software: Yes
       doc: https://secure.logmein.com/welcome/webhelp/EN/Pro/LogMeIn/c_common_Security_TwoStep.html
 
+    - name: LabTech Control Center
+      url: http://www.labtechsoftware.com/
```

Comment:
Seems like they support https. Please change the URL to `https://www.labtechsoftware.com/` :wink: